packages/garbo/src/familiar/barfFamiliar.ts

```diff
@@ -63,7 +63,7 @@ const SPECIAL_FAMILIARS_FOR_CACHING = new Map<
     $familiar`Chest Mimic`,
     {
       extraValue: ({ famexp }) =>
-        (famexp * EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")) / 50,
+        (famexp * MEAT_TARGET_MULTIPLIER() * get("valueOfAdventure")) / 50,
```

Comment:
Possibly not something for this PR, but do we need to make some adjustment here for if the copy target is not a meat_target